src/network/networkctl.c

```diff
@@ -1172,6 +1189,61 @@ static int link_status_one(
                         return r;
         }
 
+        if (info->has_ethtool_link_info) {
+                const char *duplex = duplex_to_string(info->duplex);
+                const char *port = port_to_string(info->port);
+
+                if (IN_SET(info->autonegotiation, AUTONEG_DISABLE, AUTONEG_ENABLE)) {
+                        r = table_add_cell(table, NULL, TABLE_EMPTY, NULL);
+                        if (r < 0)
+                                return r;
+                        r = table_add_cell(table, NULL, TABLE_STRING, "Auto negotiation:");
+                        if (r < 0)
+                                return r;
+                        r = table_add_cell(table, NULL, TABLE_BOOLEAN, &info->autonegotiation);
+                        if (r < 0)
+                                return r;
+                }
+
+                if (info->speed > 0) {
+                        char buf[FORMAT_BYTES_MAX];
+
+                        r = table_add_cell(table, NULL, TABLE_EMPTY, NULL);
+                        if (r < 0)
+                                return r;
+                        r = table_add_cell(table, NULL, TABLE_STRING, "Speed:");
+                        if (r < 0)
+                                return r;
+                        r = table_add_cell_stringf(table, NULL, "%sbps", format_bytes_full(buf, sizeof(buf), info->speed, false, false, false));
```

Comment:
a comment here that mentions that we are a tiny bit dirty here by formatting a bits value with a function called "bytes" might be good.

idea: maybe introduce a new cell type for bps value to cover this?

Reply:
Sorry, I cannot what you mean. Could you elaborate more?

Reply:
the "bps" value you print is in *bits* per second, but format_bytes_full() has "bytes" in the name. The fact that the difference here doesn't really matter is something to mention in a comment i think, because it might trip people up that we are not careful about the unit here, and ignore the factor of 8.

And the idea was to maybe extend format-table.c with a new TABLE_BPS cell type that takes an uint64_t or so and formats it nicely as a bps value. I mean, bps is a really generic unit, and hence we might as well support it natively in format-table.c, like we already support TABLE_SIZE (which is in bytes) or TABLE_TIMESPAN and friends